manual/source/classes/contributed_definitions/optical-spectroscopy-structure.rst

```diff
@@ -1,71 +1,81 @@
 
-    :ref:`NXopt`:
-       A generic application definition for optical spectroscopy measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
+    :ref:`NXoptical_spectroscopy`:
+       A generic application definition for optial spectorscopy measurements. This including specifically ellipsometry and Raman spectroscopy measurements, but as well other techniques such as photoluminescence, transmission, reflection measurements. The requirements are: (i) an incident photon beam, (ii) a detector to measure scattered/emitted photons and (iii) a sample.
 
     :ref:`NXellipsometry`:
-       An application definition for ellipsometry measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
+       An application definition for ellipsometry measurements, including complex systems up to variable angle spectroscopic ellipsometry.
+
+    :ref:`NXraman`:
+       An application definition for Raman spectroscopy measurements.
 
 
 Base Classes
 ------------
 
 This is the set of base classes for describing an optical experiment.
 
-    :ref:`NXbeam_path`
-       A beam path consisting of one or more optical elements.
-       
-       NXbeam_path is used in NXopt to describe the beam path, i.e. the arrangement
-       of optical elements between the excitation source and the sample, or between
-       the sample and the detector unit.
-              
-    :ref:`NXbeam_splitter`
-       A beam splitter, i.e. a device splitting the light into two or more beams.
-       
-       Use two or more NXbeam_paths to describe the beam paths after the beam
-       splitter. In the dependency chain of the new beam paths, the first elements
-       each point to this beam splitter, as this is the previous element.
+    :ref:`NXbeam`
+      Beam properties such as intensity, polarization, wavelength or direction.
 
-    :ref:`NXfiber`
-       An optical fiber, e.g. glass fiber.
+    :ref:`NXdetector`
+      A detector for signal detection.
 
-    :ref:`NXlens_opt`
-       Description of an optical lens.
-       
-    :ref:`NXpolarizer_opt`
-       An optical polarizer.
+    :ref:`NXsource`
+      A light source such as laser, lamp or LED.
 
+    :ref:`NXmonochromator`
+      A monochromator is often used to energetically disperse the scattered or emitted light.
+
+    :ref:`NXoptical_lens`
+      Description of an optical lens.
+
+    :ref:`NXoptical_window`
+      Description of an optical window.
+       
     :ref:`NXwaveplate`
-       A waveplate or retarder.
+      A waveplate or retarder.
 
-    :ref:`NXenvironment`
-       Specify external parameters that have influenced the sample,
-       such as the surrounding medium, and varied parameters e.g.
-       temperature, pressure, pH value, optical excitation etc.
+    :ref:`NXsensor`
+      Specify external parameters that have influenced the sample such as
+      varied parameters e.g. temperature, pressure, pH value, beam intensity, etc.
```

Comment:
Is the material below this about dispersive material and formula grammar to be considered too?